hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/downloader/NodeSignatureVerifier.java

```
@@ -85,7 +98,14 @@ public void verify(Collection<FileStreamSignature> signatures) throws SignatureV
         }
 
         for (FileStreamSignature fileStreamSignature : signatures) {
-            Pair<byte[], byte[]> hashAndSig = Utility.extractHashAndSigFromFile(fileStreamSignature.getFile());
+//            Pair<byte[], byte[]> hashAndSig = Utility.extractHashAndSigFromFile(fileStreamSignature.getFile());
+            Pair<byte[], byte[]> hashAndSig = null;
+            try {
+                hashAndSig = signatureFileReader
+                        .read(new BufferedInputStream(new FileInputStream(fileStreamSignature.getFile())));
+            } catch (FileNotFoundException e) {
```

On other PR I mentioned a `Utility.openQuietly(File)` that might be useful to add to not cause IOException be handled.

---

Added this method, had to rework the exception handling as `extractHashAndSigFromFile` was swallowing every type of exception and returning null values, which doesn't work as well with the refactoring.